demos/m5/README.md

```diff
@@ -2,9 +2,19 @@
 
 This directory contains the resources needed for running the release demo for Milestone 5.
 
-## Environments
+# Bootstraping
 
-This demo can run in two different environments:
+After bootstrapping the cluster with App Studio, you can prepare the demo by running the [boostrap.sh](demos/m5/boostrap.sh) script.
+```
+$ git clone https://github.com/redhat-appstudio/release-service.git
+$ cd release-service/demos/m5
+$ ./bootstrap.sh
+```
+This will create all the required resources to run the demo. Those are:
+* Demo resources in the base directory.
+* Quay secret to pull and push images (a password to the robot account will be requested).
+* Cosign secret with the public key extracted from tekton-chains.
 
-* **dev**: Use this environment to run on an empty CRC cluster with Openshift Pipelines preinstalled.
-* **staging**: This environment is meant to be used in a cluster bootstrapped with infra-deployments.
+# Triggering the demo
+
+To trigger the demo, the only thing needed after bootstraping is to apply the [release.yaml](demos/m5/release.yaml) file, which will create a new Release in the cluster.
```

Comment:
One last thing. There are two p's in bootstrapping

Reply:
I thought that it was written like that. My fault!